src/app/home/panel/navbar/navbar.component.scss

```diff
@@ -0,0 +1,50 @@
+@import "../../../../assets/styles/colors";
+
+:host {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  flex-direction: column;
+}
+
+.navbar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 99.99%;
+  height: 60px;
+  position: fixed;
+  max-width: 26rem;
+  bottom: 0;
+  border: 1.4px solid $medium-gray;
```

Comment:
Do not use fractions on pixels. Screens can only display pixels or not, there is no such thing as 0.4 of a pixel.

Reply:
ok